fvm/blueprints/fees.go

```diff
@@ -46,18 +46,17 @@ func DeployStorageFeesContractTransaction(service flow.Address, contract []byte)
 		AddAuthorizer(service)
 }
 
-const setupFeesTransactionTemplate = `
+const setupParametersTransactionTemplate = `
 import FlowStorageFees, FlowServiceAccount from 0x%s
 
-transaction(transactionFee: UFix64, accountCreationFee: UFix64, minimumStorageReservation: UFix64, storageMegaBytesPerReservedFLOW: UFix64, restrictedAccountCreationEnabled: Bool) {
+transaction(accountCreationFee: UFix64, minimumStorageReservation: UFix64, storageMegaBytesPerReservedFLOW: UFix64, restrictedAccountCreationEnabled: Bool) {
     prepare(service: AuthAccount) {
         let serviceAdmin = service.borrow<&FlowServiceAccount.Administrator>(from: /storage/flowServiceAdmin)
             ?? panic("Could not borrow reference to the flow service admin!");
 
         let storageAdmin = service.borrow<&FlowStorageFees.Administrator>(from: /storage/storageFeesAdmin)
             ?? panic("Could not borrow reference to the flow storage fees admin!");
 
-        serviceAdmin.setTransactionFee(transactionFee)
```

Comment:
I'm not sure if I understand why this has been added here, considering this is only activated when we create a new account?